scala/integration/bsp/src/org/jetbrains/plugins/scala/bsp/flow/open/ScalaBspDetailsConnectionGenerator.scala

```diff
@@ -0,0 +1,113 @@
+package org.jetbrains.plugins.scala.bsp.flow.open
+
+import com.intellij.openapi.observable.properties.ObservableMutableProperty
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.vfs.VirtualFile
+import org.jetbrains.plugins.bsp.extension.points.BspConnectionDetailsGeneratorExtension
+import org.jetbrains.plugins.bsp.flow.open.BuildToolId
+import org.jetbrains.plugins.bsp.flow.open.wizard.{ConnectionFileOrNewConnection, ImportProjectWizardStep}
+import org.jetbrains.plugins.bsp.protocol.connection.BspConnectionDetailsGenerator
+import org.jetbrains.plugins.scala.bsp.config.ScalaPluginConstants
+import org.jetbrains.plugins.scala.bsp.{BspFeatureFlags, BspPluginTemplates}
+
+import scala.jdk.CollectionConverters._
+import java.io.OutputStream
+import java.io.File
+import java.nio.file.{Files, Path, Paths, StandardOpenOption}
+import java.util
+import coursier.core.{Dependency, Module, ModuleName, Organization}
+
+import scala.concurrent.duration.Duration
+import scala.concurrent.{Await, ExecutionContext}
+import scala.collection.immutable.{HashMap, List}
+
+private object DefaultProjectViewFile {
+  val DEFAULT_PROJECT_VIEW_FILE_NAME = "projectview.scalaproject" // TODO: is this the right name?
```

Comment:
Remove this TODO, better to fix implementation which doesn't work than forgetting to remove the TODO.